Source/JavaScriptCore/b3/testb3_3.cpp

```diff
@@ -1690,7 +1690,11 @@ void testClzArg64(int64_t a)
     Value* argument = root->appendNew<ArgumentRegValue>(proc, Origin(), GPRInfo::argumentGPR0);
     Value* clzValue = root->appendNew<Value>(proc, Clz, Origin(), argument);
     root->appendNewControlValue(proc, Return, Origin(), clzValue);
+#if OS(WINDOWS)
+    CHECK_EQ(compileAndRun<unsigned>(proc, a), __lzcnt64(a));
+#else
     CHECK(compileAndRun<unsigned>(proc, a) == countLeadingZero(a));
+#endif
```

Comment:
We should make `countLeadingZero` work on Windows too.

Reply:
And we should explore why this is not correctly working on Windows.

Reply:
> The countLeadingZero implementation for testClz* tests had an off-by-one
> error on Windows - using builtin __lzcnt64 and __lzcnt instead.

Using `__lzcnt64` etc. is not right. We should know why this does not work on Windows, and we need to fix it.

Reply:
Fixed - inside countLeadingZero we were left shifting a long (1L) instead of an unsigned long long (1ull) by the bitcount.

On debug builds this was using a 32 bit signed long, on release builds a 64 bit signed long. That gave us the off-by-one on release builds.